### DISCUSSION TITLE
Prevent access to azuracast from the server public ip

### DISCUSSION THREAD

**Discussion by freqmand** [AzuraCast/AzuraCast#5064] [Q&A]
Hey community!

After setup a domain for my station, the station page still reachable from the server public ip address. 
how to prevent this and keep the station available only from the domain name ? 

Another question, is there a way to change the default login page url (/login and /dashboard) to something else (example /hiddenLogin) , to protect the login page?

Thanks.

Edit : I'm using Docker installation

**Comment by dot357** [comment]
Any answers?

**Comment by Mozitux** [comment]
Perhaps by changing the ports and using a reverse proxy such as Apache with port forwarding for the domain name.

**Comment by JavierChaparroMX** [comment]
I'm interested in this subject, I'm experimenting with adding a second interface to my ubuntu virtual machine, and I was hoping I could choose the ip/port for each component of the server to be exposed through. please let me know if anybody has figured this out.
